.github/workflows/release.yaml

```diff
@@ -0,0 +1,72 @@
+name: Build web5_cli release artifacts
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_cli_aarch64-apple-darwin:
+    runs-on: macos-latest
+    name: aarch64-apple-darwin
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        run: rustup toolchain install stable
+      - name: cargo build
+        run: |
+          cargo build --target aarch64-apple-darwin --release --package web5_cli
+          cp -v target/aarch64-apple-darwin/release/web5_cli web5_cli-aarch64-apple-darwin
+      - name: upload release asset
+        uses: AButler/upload-release-assets@v3.0
```

Comment:
was thinking about using [the GitHub native action](https://github.com/actions/upload-release-asset?tab=readme-ov-file) for this, but looks like it's no longer maintained. however they do recommend this one https://github.com/softprops/action-gh-release. any thoughts on this vs the AButler one you have here?

Reply:
I didnt see a way to use the AButler one to update an existing release. It wanted to be the one to create the release.